modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -426,7 +422,7 @@ PERF_TEST_P_(DivRCPerfTest, TestPerformance)
     }
 
     // Comparison ////////////////////////////////////////////////////////////
-    EXPECT_TRUE(cmpF(out_mat_gapi, out_mat_ocv));
+    // FIXIT unrealiable check: EXPECT_TRUE(cmpF(out_mat_gapi, out_mat_ocv));
```

Comment:
No new keywords please, we mainly use FIXME or TODO, but not FIXIT.

The idea is to grep by the keyword one day and fix what's found. Not sure anybody remembers about `FIXIT` at that point.

Reply:
Also, what has started failing? Does it fail in accuracy tests too?

Reply:
The point is `FIXIT` is used all over the perf_tests files, so I just wanted to keep consistency. Should I change all the `FIXIT` -> `FIXME`?

Reply:
The comparison is unreliable due to potential division by zero. In accuracy tests I fixed it so you can see it in `gapi_core_tests_inl.hpp` and `gapi_operators_tests_inl.hpp` (you started a conversation at one of those spots), debugged and tested it to be sure the tests don't fail
It has to be done for performance tests too, but I chose to put this task to backlog as it is not so important to check the accuracy of the performance tests when there are reliable accuracy tests themselves

Reply:
AFAIR OpenCV divides by zero safely, so we should follow.

Reply:
Despite the fact OCV backend follows OpenCV and doesn't cause any accuracy problems, Fluid and GPU backends diverge from OpenCV, so these workarounds are necessary

Reply:
Got it